### DISCUSSION TITLE
How many Dub links have you created so far?

### DISCUSSION THREAD

**Discussion by fmerian** [dubinc/dub#219] [Show and tell]
Happy Friday ☀️

I recently hit a new milestone: 100 Dub links 🙌 

![100 Dub links](https://github.com/steven-tey/dub/assets/1890087/21931b9d-b0dc-4ed8-8899-523a842879fa)

**How many links have you created so far?**

**Comment by steven-tey** [comment]
Nice milestone! :star-struck_cat: 

I gotta catch up 😅 

![CleanShot 2023-05-26 at 10 14 30](https://github.com/steven-tey/dub/assets/28986134/bd032cb6-a504-450d-a98c-64d199be6b76)
